ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -138,7 +138,7 @@ dedupCommand
    ;
 
 sortCommand
-   : SORT sortbyClause
+   : SORT (count = integerLiteral)? sortbyClause (DESC | D)?
```

Comment:
thought (non-blocking): If we create a dedicated `nonNegativeIntegerLiteral` or `positiveIntegerLiteral`, how would the error look to users if they supply a negative? It might be safer and generally applicable?